test/errorshow.jl

```diff
@@ -277,6 +280,11 @@ let undefvar
     @test err_str == "ArgumentError: not an error"
     err_str = @except_str @assert(false) AssertionError
     @test err_str == "AssertionError: false"
+
+    err_str = @except_str WString[] UndefVarError
+    @test err_str == """`WString` has been moved to the package LegacyStrings.jl:
```

Comment:
These were moved in 0.5, so maybe the code for this should just be removed?

Reply:
Do we want to keep these tests/lines? I need an adult!

Reply:
Personally, I don't think these are needed / that useful.